apps/web/src/app/[orgShortcode]/convo/layout.tsx

```diff
@@ -318,23 +324,40 @@ export default function Layout({
       )}>
       {!isMobile && (
         <>
-          <ConvoNav
-            setShowHidden={setShowHidden}
-            showHidden={showHidden}
-          />
+          <div className="flex h-full w-full min-w-96 flex-col gap-2 p-2 pt-3 xl:col-span-1 xl:min-w-80">
+            <ConvoNavHeader
+              showHidden={showHidden}
+              setShowHidden={setShowHidden}
+            />
+            {convoList}
+          </div>
           <ChildrenWithOrgIssues>{children}</ChildrenWithOrgIssues>
         </>
       )}
 
       {isMobile &&
         (!isInConvo && !isNewPage ? (
-          <ConvoNav
-            setShowHidden={setShowHidden}
-            showHidden={showHidden}
-          />
+          <div className="flex h-full w-full min-w-96 flex-col gap-2 p-2 pt-3 xl:col-span-1 xl:min-w-80">
+            <ConvoNavHeader
+              showHidden={showHidden}
+              setShowHidden={setShowHidden}
+            />
+            {convoList}
+          </div>
         ) : (
           <ChildrenWithOrgIssues>{children}</ChildrenWithOrgIssues>
         ))}
     </div>
   );
 }
+export default function Layout({ children }: { children: React.ReactNode }) {
+  const [showHidden, setShowHidden] = useState(false);
+  return (
+    <ConvoLayoutWrapper
+      convoList={<OrgConvoList hidden={showHidden} />}
```

Comment:
define `OrgConvoList` outside the return block, wrap in a `useMemo`/`useCallback` pass that